core/services/keeper/delegate.go

```diff
@@ -62,10 +61,7 @@ func (d *Delegate) ServicesForSpec(spec job.Job) (services []job.ServiceCtx, err
 		return nil, err
 	}
 	registryAddress := spec.KeeperSpec.ContractAddress
-
-	cfg := chain.Config()
-	strategy := txmgr.NewQueueingTxStrategy(spec.ExternalJobID, cfg.KeeperDefaultTransactionQueueDepth(), cfg.DatabaseDefaultQueryTimeout())
-	orm := NewORM(d.db, d.logger, chain.Config(), strategy)
```

Comment:
> Also in my quest to clean up TxStrategy I found that keepers.ORM doesn't actually use TxStrategy so I removed it from the file.

Reply:
yeah, we had a suspicion this wasn't being used but was on our back burner since keepers is going to be deprecated in favor of automation v2. Thanks for cleaning this up!